README.md

```diff
@@ -646,26 +646,19 @@ class PostVersion < PaperTrail::Version
 end
 ```
 
-If you only use custom version classes and don't use PaperTrail's built-in one,
-on Rails `>= 3.2` you must:
-
-- either declare the `PaperTrail::Version` class to be abstract like this (in an
-  initializer):
+If you only use custom version classes and don't have a `versions` table, you
+must let ActiveRecord know that `PaperTrail::Version` is an `abstract_class`.
 
 ```ruby
-# config/initializers/paper_trail.rb
-
-# the following line is required for PaperTrail >= 4.0.0 with Rails
-PaperTrail::Rails::Engine.eager_load!
-
-PaperTrail::Version.module_eval do
-  self.abstract_class = true
+# app/models/paper_trail/version.rb
+module PaperTrail
+  class Version < ActiveRecord::Base
+    include VersionConcern
```

Comment:
I think we want to be explicit here and have this line read `include PaperTrail::VersionConcern` just to make sure there is no confusion as to where that module comes from.